pages/fleet/[id].vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div v-if="zeppelin">
+    <div class="mb-10 mt-7 text-xs">
+      Category / <span class="font-semibold">{{ zeppelin.name }}</span>
+    </div>
+    <div class="grid grid-cols-2 gap-x-16 gap-y-5 mb-44">
+      <img
+        :src="zeppelin.image || ''"
+        :alt="zeppelin.name || ''"
+        class="w-full rounded"
+      />
```

Comment:
It will cause an error if the link will be empty. Here the best solution is using `v-if` and in `v-else` use some image placeholder